unit-test/background/reference-tests/broken-site-reporting-tests.js

```diff
@@ -62,7 +62,7 @@ for (const setName of Object.keys(testSets)) {
                 addActionRequests(test.adAttributionRequests, 'ad-attribution')
                 addActionRequests(test.noActionRequests, 'none')
 
-                breakageReportForTab({
+                await breakageReportForTab({
```

Comment:
Should we add a test that the parameter is not being sent on the first report, then it is present on the second?

Reply:
Not as a reference test as we'd be waiting about for all other implementations to ship. Let me check if we have others already.